docs/proposals/20230123-azure-service-operator.md

```diff
@@ -0,0 +1,188 @@
+---
+title: Managing Azure Resources with Azure Service Operator
+authors:
+  - "@nojnhuh"
+reviewers:
+  - "@CecileRobertMichon"
+  - "@jackfrancis"
+  - "@mattchr"
```

Comment:
You missed an `h` in my gh username, it's matthchr

Reply:
Sorry! Fixed.